content/applications/finance/payment_acquirers/stripe.rst

```diff
@@ -171,3 +171,18 @@ listed, you don't have anything to do.
 
 .. seealso::
    - :doc:`../payment_acquirers`
+
+.. _stripe/express-checkout:
+
+Using Express checkout
+======================
+
+Configure Apple Pay
+-------------------
+
+To enable the Apple Pay button with Stripe, you have to register your website on Stripe. To do so,
+go to :menuselection:`Payment Acquirers --> Stripe` and check that the acquirer is **enabled**. If
+it's the case, a **register your domain** button should appears. Click on it to register your domain.
+
+.. note::
+   You have to repeat this operation if you change your domain name later.
```

Comment:
```suggestion
   You have to repeat this operation whenever your domain name change.
```
It feels more natural to me, but I might be mistaken... :shrug:

Reply:
```suggestion
   You have to repeat this operation whenever your domain name changes.
```
Missing -s